docker-compose.yml

```diff
@@ -10,6 +10,8 @@ services:
     volumes:
     - ${DESECSTACK_WWW_CERTS}:/etc/ssl/private:ro
     - ./www/html:/usr/share/nginx/html:ro
+    environment:
+    - DESECSTACK_DOMAIN
```

Comment:
This variable is undocumented. I seems that it is only needed for the nginx configuration introduced in https://github.com/desec-io/desec-www/pull/5.

- Doesn't this overlap with `DESECSTACK_API_ALLOWED_HOSTS`? Why not stick to just one variable? (We can rename the latter if that would be more appriopriate.)
- If there's a good reason to have two variables, let's add documentation in both the `README.md` and `.env.default` files.